docs/developing-in-the-cloud/cloud-dev-environments/aws-ecs.mdx

```diff
@@ -3,5 +3,53 @@ title: "AWS Elastic Container Service"
 sidebarTitle: "AWS ECS"
 ---
 
-You can refer to the instructions found [here](/production/deploying-to-cloud/aws/setup).
```

Comment:
This is not the right doc. This doc is for multi-dev environment on cloud.

You should put the architecture doc in this folder: https://github.com/mage-ai/mage-ai/tree/master/docs/production/deploying-to-cloud/aws